build-system/tasks/e2e/mocha-ci-reporter.js

```diff
@@ -28,6 +29,7 @@ function ciReporter(runner, options) {
   Base.call(this, runner, options);
   this._mochaDotsReporter = new MochaDotsReporter(runner, options);
   this._jsonReporter = new JsonReporter(runner, options);
+  this._mochaJunitReporter = new MochaJUnitReporter(runner, options);
```

Comment:
Clarifying question: Is this the change that was requested by https://github.com/ampproject/amphtml/pull/33682#discussion_r611716106?

Reply:
Yes